superset/assets/javascripts/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -6,7 +6,7 @@ import * as Actions from '../actions';
 import SqlEditor from './SqlEditor';
 import CopyQueryTabUrl from './CopyQueryTabUrl';
 import { areArraysShallowEqual } from '../../reduxUtils';
-import { getParamFromQuery } from '../../../utils/common';
+import URI from 'urijs';
```

Comment:
I really need it for this PR though: https://github.com/airbnb/superset/pull/2537 

We should dig into what is taking those 13 mb. I'll try to dig into that, I bet urijs doesn't really weight in..